install/kubernetes/cilium/charts/operator/values.yaml

```diff
@@ -1,4 +1,11 @@
-image: operator
+# Include the cilium-operator Deployment
+enabled: true
+
+image:
+  repository: cilium-operator
+  tag: latest
```

Comment:
cc @joestringer do you thing the `examples/install/Makefile` will catch this in the release process?

Reply:
Looks like we'll need to amend the Makefile to look at this file in addition to the global values.yaml. I can prep a patch for that.

Reply:
Pushed.